.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "sqflint"]
+	path = sqflint
+	url = https://github.com/RZSenfo/sqflint
```

Comment:
I'd use my version of sqflint for this repo? I'm not sure how to do this tho

Reply:
Will do,
was mainly still there because I used my fork for development.
Probably need to merge sqflint before the extension